Sources/ConnectionPoolModule/PoolStateMachine.swift

```diff
@@ -88,8 +88,8 @@ struct PoolStateMachine<
         case runKeepAlive(Connection, TimerCancellationToken?)
         case cancelTimers(TinyFastSequence<TimerCancellationToken>)
         case closeConnection(Connection, Max2Sequence<TimerCancellationToken>)
-        case shutdown(Shutdown)
-
+        case initiateShutdown(Shutdown)
+        case shutdown([TimerCancellationToken])
```

Comment:
what is the difference between `initiateShutdown` and `shutdown`? can we add a developer comment here please?

Reply:
Wanna rename this to `cancelEventStreamAndFinalCleanup`?